controllers/openshift_controller.go

```diff
@@ -447,6 +448,12 @@ func (r *KataConfigOpenShiftReconciler) setRuntimeClass() (ctrl.Result, error) {
 				Name: runtimeClassName,
 			},
 			Handler: runtimeClassName,
+			Overhead: &nodeapi.Overhead{
+				PodFixed: corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("250m"),
```

Comment:
Maybe it would be a good idea to add a comment here explaining where these values came from to help future readers understand the rationale behind setting those values. You have already mentioned that you got those values from, https://github.com/kata-containers/packaging/blob/master/kata-deploy/k8s-1.18/kata-runtimeClasses.yaml#L7, in the PR description, so I would suggest add that info here as in the form a comment too.

Reply:
Yes, it's a good idea. I've done it and pushed